Cargo.toml

```diff
@@ -10,6 +10,7 @@ serde_macros = ["webrender_traits/serde_macros"]
 [dependencies]
 app_units = "0.2.5"
 bit-set = "0.4"
+bincode = "0.5.9"
```

Comment:
@nox Is introducing this dependency likely to cause us any problems with WR + CI issues?